salt/states/boto_secgroup.py

```diff
@@ -379,7 +379,7 @@ def _get_rule_changes(rules, _rules):
     return (to_delete, to_create)
 
 
-def _rules_present(name, rules, vpc_id=None, vpc_name=None,
+def _rules_present(name, rules=None, vpc_id=None, vpc_name=None,
```

Comment:
I believe this was a hasty change I made when testing deleting and I believe i based it off the kwarg for boto_secgroup.present that does default rules to None. I'll run some tests again without to change to see if i can reproduce it.. otherwise might just revert this change along with the egress..